### ISSUE TITLE
Remove or disable all unstable tests.

### ISSUE THREAD

**Issue by TennyZhuang** [tikv/tikv#11177]
There are too many unstable tests in TiKV and not fixed for too long time. A part of them were listed in #10506 , but not all.

E.g. in #11095 , a simple patch, some unrelated test failed 4 times unexpectedly. The unstable tests waste both CI resources and **the time of developers** who should not be responsible for these bugs. Please note that they can do nothing to handle the failure except retyping /run-tests , which is really meaningless.

Another problem is that, the unstable tests are really confusing for new contributors from community. They may waste much time to debug their codes for non-existent bugs.

Why not just remove them or disable them in master, and add them back with new pull requests? For tracking, we can assign every unstable test to specific developer, only these developers spend time on them. We can also create a new branch instead of master to run these tests periodically. Of course, we can still track them with tracking issue.

@zhouqiang-cl @BusyJay @tisonkun @breeswish

**Comment by BusyJay** [comment]
I'm working on some of the cases and also assigning some of them to owners. Hope to make most of them stable in short time.

Some test cases involve heavy IO/CPU operations, perhaps using memory disk/high end disk can also help improve the success rate.

**Comment by TennyZhuang** [comment]
> I'm working on some of the cases and also assigning some of them to owners. Hope to make most of them stable in short time.
> 
> Some test cases involve heavy IO/CPU operations, perhaps using memory disk/high end disk can also help improve the success rate.

Please note that this issue is not about these unstable tests per se, but about how to deal with them. Any test that might cause the master ci to fail should be rolled back or disabled as soon as possible, rather than hanging around causing problems for others and waiting for someone to fix it.

**Comment by TennyZhuang** [comment]
Boosting the fix speed did not solve the issue

**Comment by BusyJay** [comment]
I agree new test cases should be rollback or fixed as soon as possible, but I don't think it's appropriate to disable unstable test cases. Even test cases are unstable, they are still value at covering some part of the system. Disabling it or removing it will lose the opportunity to find errors early.

If there are too many unstable test cases, it means the code quality becomes low, we need to pause and find out what's going wrong instead of producing more low quality code.
